### DISCUSSION TITLE
Brand Analytics Role Access

### DISCUSSION THREAD

**Discussion by AppDev2020-admin** [amzn/selling-partner-api-models#3512] [General]
I have been requesting Brand Analytics Role access for weeks and keep getting rejected. They say my website 'should display information about function details, and other features of your app'. From my end I have tried including all relevant information, still my request is getting rejected. Can someone who has successfully received this access share what was their approach or share an example app website which was successful in convincing the team to grant them access?

**Comment by aluminum-ice** [comment]
I have the same exact problem. We are a new start-up so our product doesn't yet exist, which confuses me because it isn't clear what information they want to see on our website for a product that does _not_ yet exist. I keep getting this generic rejection email:

> Hello,
> 
> Thank you for updating your Amazon Selling Partner API (SP-API) Developer Profile.
> 
> We have completed our assessment and have determined that you do not meet criteria for Brand Analytics role access on SP-API. To protect Amazon Customers, we require Developers to have an online business website with publicly available, descriptive information about the analytical products and services provided for Amazon Selling Partners. Services and product must be consistent with section 4.4 and 4.5 of our Acceptable Use Policy.
> 
> No changes have been made to your SP-API access at this time.
> 
> Case Appeals：
> If you would like to appeal the rejection of Brand Analytics role, please do not reopen this case. Please update your Developer Profile to create a new case. Your update should include a website URL that provides details about the services your application offers to Amazon Sellers. The website should display information about function details, and other features of your app.

I added high-level details about the products we will build, but without building them we can't provide specific details. It seems like a catch-22 without helpful information on why its getting rejected.

**Reply by AppDev2020-admin** [reply]
Same with us. We keep getting similar rejection emails with very less constructive feedback on what we can do to get the access. This has been a very frustrating experience. It would be great to learn from someone who was successful in getting this access.

**Comment by 0xs3n4** [comment]
i have the same problem... Anyone for help us?

**Comment by sungolivia** [comment]
hey everyone, from what I have seen, having like a public website, with clear description about the product and services provided should help. But at the same time, if you can paste what's unclear or less constructive, I can provide the feedback about the unclear responses to our team! Not part of the team that looks through but will try to help where I can.

**Reply by 0xs3n4** [reply]
We are providing the public website and etc... But first we would like to have access to the functionalities to be able to test them beforehand, and even validate the business rule of our product, because we are thinking of doing something that sometimes the API does not yet provide.
